### PR TITLE
fix: Puppeteer stringified button click type

### PR DESCRIPTION
**Pull request by adamraine**
I noticed there were missing quotes around the click type in the puppeteer stringify extension.

### CODE REVIEW

File: src/PuppeteerStringifyExtension.ts [hunk 1]
```diff
@@ -63,7 +63,10 @@ export class PuppeteerStringifyExtension extends StringifyExtension {
     for (const line of helpers.split('\n')) {
       out.appendLine(line);
     }
-    out.endBlock().appendLine('})();');
+    out.endBlock().appendLine('})().catch(err => {').startBlock();
+    out.appendLine('console.error(err);');
+    out.appendLine('process.exit(1);');
+    out.endBlock().appendLine('});');
```
Comment (adamraine) [inline src/PuppeteerStringifyExtension.ts:69]: Drive-by exit 1 when an error is thrown

**Review by OrKoN — APPROVED** [review]
